quickstarts/microsoft.logic/logic-app-create/metadata.json

```
@@ -10,14 +10,37 @@
   "type": "QuickStart",  
   "tags": {
     "subscriptiontype": "Azure4Student",
+    "id": 21,
+    "rank": 7,
     "resource": "Logic Apps, MySQL",
     "os": "Linux, Logic App",
     "level": "beginner",
     "cost": "$",
     "scenario": "Azure Logic Apps",
     "roles": "Developer",
     "learning": "https://docs.microsoft.com/en-us/learn/browse/?products=azure-logic-apps&WT.mc_id=Portal-Microsoft_Azure_Education",
-    "scenarioLink": "https://azure.microsoft.com/en-us/services/logic-apps/",
-    "isActive": "true"
+    "isA4EActive": "true",
+    "usefulLink": [
+            {
+                "name": "Getting Started",
+                "rank": 1,
+                "url": "https://azure.microsoft.com/en-us/resources/templates/logic-app-create/"
+            },
+            {
+                "name": "Azure Logic App",
```

Official product name is "Azure Logic Apps":

```suggestion
                "name": "Azure Logic Apps",
```